modules/javafx.graphics/src/main/java/com/sun/glass/ui/mac/MacView.java

```diff
@@ -78,6 +78,7 @@ static int getMultiClickMaxY_impl() {
     @Override native protected boolean _enterFullscreen(long ptr, boolean animate, boolean keepRatio, boolean hideCursor);
     @Override native protected void _exitFullscreen(long ptr, boolean animate);
     @Override native protected void _enableInputMethodEvents(long ptr, boolean enable);
+    @Override native protected void _finishInputMethodComposition(long ptr);
```

Comment:
Now that you override this method on Mac, I recommend changing the comment in the base class from `... needed only on Windows` to `... needed only on some platforms`.

Reply:
I'll pick up the comment change in the PR #1351.